drivers/flash/spi_flash_sfdp.c

```diff
@@ -548,11 +603,11 @@ static int spi_flash_init(struct device *dev)
 
 	k_sem_init(&data->sem, 1, UINT_MAX);
 	dev->driver_api = &spi_flash_api;
-	return 0;
+	return r;
 }
 
 static struct spi_flash_data spi_flash_memory_data;
 
-DEVICE_INIT(spi_flash_memory, "flash0", spi_flash_init,
-	    &spi_flash_memory_data, NULL, POST_KERNEL,
-	    CONFIG_SPI_FLASH_SFDP_INIT_PRIORITY);
+DEVICE_INIT(spi_flash_memory, CONFIG_SPI_FLASH_DRV_NAME,
+			spi_flash_init, &spi_flash_memory_data, NULL, POST_KERNEL,
```

Comment:
indentation issue: keep lines aligned to latest '(' + 1 space, as it used to be actually.

Reply:
The original file uses ‘\t’ + space.
There are different indentations in my editor. 😭

Reply:
try to see if your editor can be configured to apply linux code style